### ISSUE TITLE
Typo(s) in the software list (according to a native U.S. English speaker)

### ISSUE THREAD

**Issue by CtrlAltCuteness** [ROCKNROLLKID/Open-Source-Community#12]
I will let you know ahead of time that I'm only doing this in good faith and not trying to be rude or anything (also, I don't have any SVN system currently installed on my system _and_ know how to use it).

**Games and game related software:**
[Battle for Wesnoth](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/905920eda18a005b67477c4d9b7f086fd6c9668b/List%20of%20open-source%20software#L14): `elements hat runs` should be `elements that runs`
[GZDoom](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/905920eda18a005b67477c4d9b7f086fd6c9668b/List%20of%20open-source%20software#L19): `A source port engien` probably should be `A source port engine`

**Security and privacy related software:**
[OpenVPN](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/905920eda18a005b67477c4d9b7f086fd6c9668b/List%20of%20open-source%20software#L63): `An VPN service`, if I'm not mistaken, my grammar says it should be `A VPN service`
[TOR](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/905920eda18a005b67477c4d9b7f086fd6c9668b/List%20of%20open-source%20software#L68): `An a encrypted` probably drop the lonely word `a`

**Office and archive related software:**
[PeaZip](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/905920eda18a005b67477c4d9b7f086fd6c9668b/List%20of%20open-source%20software#L78): `A full-features archives` ... being late at night, and not used / researched it myself, probably needs rewording.

**Paint and art related software:**
[Krita](https://github.com/ROCKNROLLKID/Open-Source-Community/blob/905920eda18a005b67477c4d9b7f086fd6c9668b/List%20of%20open-source%20software#L111): `A scetching and` ... probably `sketching` ?

I stopped right before starting the **Miscellaneous** category because I am heavily sleepy.

**Comment by ROCKNROLLKID** [comment]
Thank. I have corrected them. If you find anymore, please re-open another ticket or a PR.
